nodejs/week3/warmupExercise/index.js

```diff
@@ -0,0 +1,58 @@
+// Homework nodejs/week3 warmup section
+
+const express = require("express");
+const app = express();
+app.use(express.urlencoded({extended: true}));
+
+app.get("/", (req, res) => res.send("Welcome to Nodejs week3 warmup homework"));
+
+// Multiplication
+// http://localhost:3000/calculator/multiply?firstParam=2&secondParam=4&thirdParam=5
+// response: Multiplication of given numbers = 40
+app.get('/calculator/multiply', (request, response) => {
```

Comment:
Hello, nice work you did here, just a small observation, on the homework description you are asked to be able to get this result `/calculator/multiply?firstParam=1&secondParam=2&secondParam=4` => 8, but you get an error, try to figure out how to solve it.